subsys/ipc/ipc_service/backends/ipc_icbmsg.c

```diff
@@ -656,7 +656,7 @@ static int match_bound_msg(struct backend_data *dev_data, size_t rx_block_index,
  *
  * @param[in] ept	Endpoint to use.
  *
- * @return		O or negative error code.
+ * @return		number of sent blocks or negative error code.
```

Comment:
similar to https://github.com/zephyrproject-rtos/zephyr/pull/73156/files#r1613043527